HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -282,4 +287,7 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     process = customizeHLTfor41632(process)
     process = customizeHLTfor42410(process)
 
+    if menuType in ["GRun","HIon","PIon","PRef"]:
+        process = customizeHLTfor42454(process)
```

Comment:
```suggestion
        process = customizefor42454(process)
```

I assume you did not test @emiglior